packages/scripts/src/upgrade-tests/voting2/1_Propose.ts

```diff
@@ -105,19 +118,28 @@ async function main() {
 
   console.log("2. CRAFTING GOVERNOR TRANSACTIONS");
 
+  const votingUpgraderValidateTxSender = await votingUpgrader.populateTransaction.canRun();
```

Comment:
`populateTransaction` seems redundant here as we need the actual call response, not its input data